### DISCUSSION TITLE
Do we need to install allure command line generator

### DISCUSSION THREAD

**Discussion by ravibabu101143** [eliasnogueira/selenium-java-lean-test-architecture#103] [Q&A]
Hi,

I am new to allure report and as per documentation, we need to download and set class path to run allure serve command in command line to generate html report. 

As per this new framework we need to take care this manually or framework will be taken care.

Thanks,
Ravi

**Comment by eliasnogueira** [comment]
Hi @ravibabu101143 ,
If you need to see the report after any test execution, you need to run the `mvn allure:serve` command.
I didn't add the automatic approach for each execution because sometimes, for local ones, you will already see the similar output in your IDE.

In the implementation the automatic download of the allure-command-line (necessary to run the `mvn allure:serve` is placed here
https://github.com/eliasnogueira/selenium-java-lean-test-architecture/blob/master/pom.xml#L149-L182

If the report is not being generated in the path defined you can create a property file in the `src/test/java` named `allure.properties` with the same content of this file
https://github.com/eliasnogueira/restassured-complete-basic-example/blob/master/src/test/resources/allure.properties

**Reply by ravibabu101143** [reply]
Thank you @eliasnogueira .
